definitions/linux.yaml

```diff
@@ -345,6 +358,16 @@ sources:
 labels: [Configuration Files, System]
 supported_os: [Linux]
 ---
+name: ListProcessesPs
```

Comment:
Lets call it ListProcessesPsCommand so it's easily differentiated from in the GRR action one.